gomanifest/internal/manifest_test.go

```diff
@@ -152,5 +157,5 @@ func TestSaveManifest(t *testing.T) {
 	assert.Equal(t, nil, err)
 
 	// Read output json and check for its size
-	assert.Equal(t, 144, len(manifestContent), "Output manifest file size missmatch")
+	assert.Equal(t, 117, len(manifestContent), "Output manifest file size missmatch")
```

Comment:
Please check whether replace really replaced the mod by checking the value, not just numbers.

Reply:
Fixed in new code, added separate function to verify replace directive case.